docs/book/src/developers/releasing.md

```diff
@@ -95,7 +95,13 @@ This will automatically trigger a [Github Action](https://github.com/kubernetes-
   - `export GITHUB_TOKEN=<your GH token>`
   - `make promote-images`
 
-This will automatically create a PR in [k8s.io](https://github.com/kubernetes/k8s.io) and assign the CAPZ maintainers. Example PR: https://github.com/kubernetes/k8s.io/pull/3007.
+This will automatically create a PR in [k8s.io](https://github.com/kubernetes/k8s.io) and assign the CAPZ maintainers. Example PR: https://github.com/kubernetes/k8s.io/pull/4284.
+
+For reviewers of the above-created PR, to confirm that the resultant image SHA-to-tag addition is valid, you can check against the Google Container Registry.
```

Comment:
should we make it clear that this isn't a required step for review given that the SHA is auto-generated by `kpromo`? If the tool ever produces a PR that has a SHA mismatching the tag that would be a bug, not human error.

Reply:
What if someone ran `make promote-images`, then later on before the image PR is merged, found out that the release was tagged to the wrong commit, deleted the tag, then re-created against a new commit?

Reply:
fair :)